check_dirty.py

```diff
@@ -9,7 +9,9 @@
 def check_if_dirty(repo_path):
     repo = Repo(repo_path)
     if repo.is_dirty():
-        print("Repo is dirty.", file=sys.stderr)
+        print("Repo is dirty: ", repo_path, file=sys.stderr)
```

Comment:
I'm surprised printing the repo path this way works, but I checked the reference documentation and it makes sense. 👍